test/support/commands.js

```diff
@@ -2,3 +2,11 @@ Cypress.Commands.add('getByTestId', testId => cy.get(`[data-testid="${testId}"]`
 Cypress.Commands.add('getByTitle', title => cy.get(`[title="${title}"]`));
 Cypress.Commands.add('getTooltip', () => cy.get('[role="tooltip"]'));
 Cypress.Commands.add('getByAriaLabel', ariaLabel => cy.get(`[aria-label="${ariaLabel}"]`));
+
+// Annotations-specific commands
+Cypress.Commands.add('clickAnnotationLinkByComment', comment => {
```

Comment:
These commands seem to be very general, should this be organized in a separate file like `annotations.js` in this directory and imported? Or maybe just define the helper function locally in `Annotations.e2e.test.js`